test/powershell/Modules/PackageManagement/Nuget.Tests.ps1

```diff
@@ -1321,7 +1321,9 @@ Describe Get-PackageSource -Tags "Feature" {
         $a | where { $_.Name -eq 'jQuery'  } | should be $true
     }
    
-    It "get-packageprovider--find-package, Expect succeed" {
+    # travisez13 201608030 failing when enabling daily test run
```

Comment:
Resolved

---
In reply to: [77046872](https://github.com/PowerShell/PowerShell/pull/2139#discussion_r77046872) [](ancestors = 77046872)